content/blog/20240624-practical-ard.md

```diff
@@ -0,0 +1,90 @@
+
+Ultimately, we should aim to make the ARD specification as complex as needed but as simple as possible. We also need to be careful to not have too many optional requirements and instead prioritize requirements that bring the most value to most users.
+
+### Right-sizing the CEOS-ARD building blocks
+
+CEOS-ARD is already somewhat split into building blocks, which are good enough for the purpose of CEOS-ARD. In general, however, they are too broadly scoped.
+
+For example, one building block aims to cover all "product metadata" rather than discrete metadata about the capturing satellite or projection information. Ideally, CEOS-ARD would be defined in smaller blocks that the individual product family specifications can pick from, minimizing the development time of new product family specifications.
+
+These building blocks could even match STAC extensions, but that might be difficult in some cases due to different scoping (STAC was primarily designed for search and discovery) and due to the fact that they are cleanly discretized (more on that below).
+
+Another reason to split into smaller building blocks is that it allows smaller groups of experts to more easily work on individual building blocks. Limiting the scope ensures that the work is still manageable and you can get to a result in a reasonable amount of time. I believe we should aspire to follow the best practices of categorization / discretization as defined by [Peter Strobl (JRC)](https://it.linkedin.com/in/peter-strobl-79b7a191):
+
+> - **assessable**: all characteristics used for distinguishing categories must be identified, and objective/measurable
+> - **unambiguous**: categories mutually exclude each other
+> - **gap free**: each item can be assigned to a specific category
+> - **intrinsic**: assignment of an item to a category is independent of other, not previously agreed characteristics
+> - **instantaneous**: assignment of an item to an category is independent of that of other items
+> - **product family specifications**: different granularity in categorization is achieved by a hierarchy
+
+Getting the categorization of the building blocks right will be very challenging, but I believe that once we have that settled it will be much easier to fill them with definitions for ARD. From there, it will be easier to evolve them into sets of building blocks that will be useful for specific types of data.
+
+## STAC supports a user-centric and practical approach
+
+Users currently seem to benefit a lot from STAC, as its standardization of metadata brings them one step closer to ARD already. CEOS-ARD has a separate XML-based metadata language, which is not ideal. I believe CEOS-ARD should be expressed in STAC, because it provides the metadata in a form that people already use and is supported by a vibrant software ecosystem.
```

Comment:
```suggestion
Users currently seem to benefit a lot from STAC, as its standardization of metadata brings them one step closer to ARD already. CEOS-ARD has a separate XML-based metadata language, which is not ideal. I believe CEOS-ARD should be expressed in STAC, because it provides the metadata in a form that people already use and is supported by a [vibrant software ecosystem](https://stacindex.org/ecosystem).
```